cl/opinion_page/views.py

```diff
@@ -948,6 +1098,159 @@ async def view_authorities(
     )
 
 
+async def check_flag_exists(flag_name: str) -> bool:
+    return await sync_to_async(
+        waffle.get_waffle_flag_model().objects.filter(name=flag_name).exists
+    )()
```

Comment:
I dont think we need this method.  There's already a setting to handle flags that aren't defined in the database. When a flag is missing, it returns the `WAFFLE_FLAG_DEFAULT` value by default. In our production environment, I believe `WAFFLE_FLAG_DEFAULT` is set to False.

Here's the Django-waffle documentation for more details on this behavior: 

https://waffle.readthedocs.io/en/stable/types/flag.html#auto-create-missing